examples/get-view-demo.js

```diff
@@ -0,0 +1,23 @@
+const { Source } = require('..')
+const rdf = require('rdf-ext')
+
+async function main () {
+  const source = new Source({
+    endpointUrl: 'https://int.lindas.admin.ch/query'
+  })
+
+  const view = await source.view(rdf.namedNode('https://ld.stadt-zuerich.ch/statistics/view/V000002'))
```

Comment:
This file is a demo showing how to read a cube-view from a source. It's not part of the lib, but an example